samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/FakeApi.java

```diff
@@ -1501,11 +1501,11 @@ private HttpRequest.Builder testEnumParametersRequestBuilder(@javax.annotation.N
   /**
    * Fake endpoint to test group parameters (optional)
    * Fake endpoint to test group parameters (optional)
-   * @param apiRequest {@link APItestGroupParametersRequest}
+   * @param apiRequest {@link APITestGroupParametersRequest}
    * @return CompletableFuture&lt;Void&gt;
    * @throws ApiException if fails to make API call
    */
-  public CompletableFuture<Void> testGroupParameters(APItestGroupParametersRequest apiRequest) throws ApiException {
+  public CompletableFuture<Void> testGroupParameters(APITestGroupParametersRequest apiRequest) throws ApiException {
```

Comment:
@Nicklas2751 fyi. i saw this change, which may become a breaking change for other users

for the time, we will consider this as a bug fix to correct the naming and see if there's a need to provide a fallback later (one way to fallback is to use customized templates)